### PR TITLE
[#5] Read k8s environment

### PR DESCRIPTION
**Pull request by tangrammer**

### CODE REVIEW

File: projects/example-lumen/api.R [hunk 1]
```diff
@@ -3,6 +3,16 @@ library(httr)
 library(jsonlite)
 library(readr)
 
+
+lumen_dataset_url<- function(lumen_instance, dataset_id){
+     lumen_domain <- "akvotest"
+     if(Sys.getenv("K8S_ENVIRONMENT") == "production"){
```
Comment (dlebrero) [inline projects/example-lumen/api.R]: Add an explicit LUMEN_DOMAIN config

Doing logic based on the environment always ends up in a big mess.

**Review by dlebrero — CHANGES_REQUESTED** [review]

File: projects/example-lumen/api.R (hunk 1 above)
Comment (tangrammer) [reply projects/example-lumen/api.R]: d6a3ca9 an attempt to fix that

**Review by dlebrero — CHANGES_REQUESTED** [review]
Approach is almost good. 

With the implemented approach to deploy the generic config changes you will need to make a change in any project folder. 

I think it makes more sense for the config to be deployed on it own. My suggestion would be for it to have its own deploy script that deploy changes in test to test and prod to prod, ignoring the branch logic, but maybe this will confuse things.

**Review by dlebrero — APPROVED** [review]
After talking with @tangrammer about all the possible issues and possible fixes, we decided that until there are several projects already running in prod, the easiest to understand is to leave things as per this PR.

**Pull request merged**